R/mcmc.R

```diff
@@ -773,6 +773,14 @@ setMethod("mcmc",
 ##
 ##' @describeIn mcmc Obtain the posterior samples for the model parameters in the
 ##' `LogisticLogNormalOrdinal`.
+##'
+##' The generic `mcmc` method returns a `Samples` object with elements of the
```

Comment:
```suggestion
##' For `LogisticLogNormalOrdinal`, the generic `mcmc` method returns a `Samples` object with elements of the
```
so that when reading the documentation page it is clear what this refers to